sdk/src/azure/storage/az_storage_blobs_blob_client.c

```diff
@@ -54,19 +54,19 @@ AZ_NODISCARD az_storage_blobs_blob_client_options az_storage_blobs_blob_client_o
 }
 
 AZ_NODISCARD az_result az_storage_blobs_blob_client_init(
-    az_storage_blobs_blob_client* client,
-    az_span uri,
+    az_storage_blobs_blob_client* out_client,
+    az_span endpoint,
```

Comment:
Good catch! I wish we had a linter tool that detected header params and source params matched. I ran one locally a while back.